src/main/java/com/shop/controller/drivers/GetAllDriversController.java

```diff
@@ -0,0 +1,25 @@
+package com.shop.controller.drivers;
+
+import com.shop.lib.Injector;
+import com.shop.model.Driver;
+import com.shop.service.DriverService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllDriversController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.shop");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> drivers = driverService.getAll();
+        req.setAttribute("drivers", drivers);
+        req.getRequestDispatcher("/WEB-INF/views/drivers/all-drivers.jsp").forward(req, resp);
```

Comment:
We were told not to use plurals in package names
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/driver/all-drivers.jsp").forward(req, resp);
```